src/components/routes/AppRoutes.tsx

```diff
@@ -56,11 +56,12 @@ const AppRoutes = () => {
       path: ROUTES.FTA + ROUTE_PARAMS.FTA_FRAGMENT,
       element: <FaultTreeDashboard />,
     },
-    {
+
+    !ENVVariable.DISABLE_FMEA && {
       path: ROUTES.FMEA,
       element: <FmeaOverview />,
     },
-    {
+    !ENVVariable.DISABLE_FMEA && {
```

Comment:
I am not sure if this can work correctly:

![image](https://github.com/user-attachments/assets/f6a42109-a564-4579-ae36-91978cd5dc4a)